web/html/wallet.js

```diff
@@ -6,6 +6,10 @@ closeModalOnClickOn("close-button");
 closeModalOnClickOn("nicknameCancelBtn");
 getWallets();
 
+function addPrefixUrl(relativeURL) {
+    return `http://my.massa/thyra/plugin/Massalabs/Massa%20Wallet/${relativeURL}`
```

Comment:
this part smells bad I would welcome your improvement suggestions

Reply:
Can't you generate that from the register function ? My point is that you are defining the author and plugin name twice...